lib/govuk_publishing_components/presenters/meta_tags.rb

```diff
@@ -27,6 +27,7 @@ def meta_tags
 
       def add_core_tags(meta_tags)
         meta_tags["govuk:format"] = content_item[:document_type] if content_item[:document_type]
+        meta_tags["govuk:publishing-app"] = content_item[:publishing_app] if content_item[:publishing_app]
```

Comment:
I wonder if it’s worth renaming this to “publishing-application” before this becomes too permanent  for consistency with the “rendering-application” meta tag.